src/pages/LandingPage.tsx

```diff
@@ -0,0 +1,90 @@
+import { Button } from '@chakra-ui/button';
+import { Image } from '@chakra-ui/image';
+import { Box, Flex, Text, VStack } from '@chakra-ui/layout';
+import styled from '@emotion/styled';
+import { useCallback, useState } from 'react';
+import { useNavigate } from 'react-router';
+import Slider from 'react-slick';
+
+import LargeLogo from '@/components/base/LargeLogo';
+import useLocalStorage from '@/hooks/useLocalStorage';
+import { LandingPageItem } from '@/utils/constants/landingPageItem';
+import ROUTES from '@/utils/constants/routes';
+
+const LandingPage = () => {
+  const navigate = useNavigate();
+  const [dragging, setDragging] = useState<boolean>(false);
+
+  const [token, _] = useLocalStorage('accessToken', '');
+
+  const onBeforeChange = useCallback(() => {
+    setDragging(true);
+  }, []);
+
+  const onAfterChange = useCallback(() => {
+    setDragging(false);
+  }, []);
+
+  const settings = {
+    dots: true,
+    infinite: true,
+    speed: 500,
+    slidesToShow: 1,
+    slidesToScroll: 1,
+    arrows: false,
+    autoplay: true,
+    autoplaySpeed: 3000,
+    pauseOnHover: true,
+    touchThreshold: 200,
+    beforeChange: onBeforeChange,
+    afterChange: onAfterChange,
+    adaptiveHeight: false,
+  };
+
+  return (
+    <Box py='4rem'>
+      <LargeLogo src='/logo-lg.svg' />
+      <Box bg='#ececec' mx='2rem' borderRadius='1rem'>
+        <StyledSlider {...settings}>
+          {LandingPageItem.map((item) => (
```

Comment:
```suggestion
          {LandingPageItem.map(({id, imageUrl, content}) => (
```

구조분해할당해서 사용하는 방법도 있을 것 같아요~👍